### DISCUSSION TITLE
Camera focus problem

### DISCUSSION THREAD

**Discussion by ManalKR** [bonsai-rx/bonsai#1663] [Q&A]
Hi,

I've developed a workflow in which I record brain activity using the Neurophotometrics system while simultaneously capturing behavior. However, I'm encountering an issue with the camera: it frequently becomes blurry before refocusing. I suspect that the movement of the optic fiber cable is causing the camera's focus to shift. I am using a Logitech webcam. Is there a method to maintain consistent focus on both the mouse and the arena, thereby preventing blurriness?

file:///Users/mmansour1/Downloads/GreenGRAB_Recording.svg

Thank you,
Manal

**Comment by bruno-f-cruz** [comment]
Heya,

You can try disabling the autofocus in the camera. Not sure how that camera works but going through CameraProperties and finding the relevant properties (e.g. Focus) might allow you to fix this behavior. Alternatively, the camera software might have a way to disable this option too.
